### PR TITLE
Update new-navbar

### PR DESCRIPTION
**Pull request by AbeRuelas**
Update NavBar component. Delete console log.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -8,30 +8,37 @@ import Home from "./components/Home";
...
+    setIsLoggedIn(true);
+  };
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+  };
 
-      <BrowserRouter>
+  return (
+    <BrowserRouter>
```
Comment (jmedina1329) [inline src/App.jsx]: What's the purpose of moving the NavBar and Footer inside of BrowserRouter?

File: src/App.jsx [hunk 2]
```diff
@@ -8,30 +8,37 @@ import Home from "./components/Home";
...
+      <Fragment>
+        <NavBar isLoggedIn={isLoggedIn} onLogout={handleLogout} />
         <Routes>
           <Route path="/" element={<Home />} />
-          <Route path="/aboutUs" element={<AboutUs />} />
+          <Route path="/about-us" element={<AboutUs />} />
           <Route path="/calendar" element={<Calendar />} />
-          <Route path="/login" element={<Login />} />
+          <Route path="/login" element={<Login onLogin={handleLogin} />} />
```
Comment (jmedina1329) [inline src/App.jsx:33]: This code should be handled by the login component that was made. We don't want the navbar to log the user in.

**Review by jmedina1329 — CHANGES_REQUESTED** [review]
Also please pull the latest code as I think this branch is out of date.

File: src/App.jsx (hunk 1 above)
Comment (umbra-codex) [reply src/App.jsx]: The NavBar has to be within the BrowserRouter component based on my research. If it's not inside the BrowserRouter component, it won't be able to properly handle the navigation links.

**Review by jmedina1329 — APPROVED** [review]

**Review by jmedina1329 — APPROVED** [review]

**Pull request merged**